src/main/resources/openapi/courtLogs.yaml

```diff
@@ -0,0 +1,79 @@
+openapi: 3.0.1
+
+info:
+
+  version: 0.0.1
+  title: Modernised DARTS
+  description: |-
+    Modernised DARTS (Digital Audio Recording and Transcription Service).
+
+servers:
+  - url: http://localhost:4550/
+
+
+paths:
+  /courtlogs:
+    get:
+      tags:
+        - CourtLogs
+      summary: An Endpoint which allows users to request courtlogs
+      description: |-
+        An Endpoint which allows users to request courtlogs using courthouse, casenumber, startdate and enddate
+      parameters:
+        - in: query
+          name: Courthouse
+          schema:
+            type: string
+        - in: query
+          name: caseNumber
```

Comment:
Maybe not a comment for this PR, but I the naming standard for query param names is `snake_case` (https://hmcts.github.io/restful-api-standards/#130).